src/App.tsx

```diff
@@ -10,7 +10,7 @@ import { BrowserRouter } from "react-router-dom";
 
 function App(): JSX.Element {
   return (
-    <BrowserRouter>
+    <BrowserRouter basename="/finesse-frontend">
```

Comment:
this is part of the reason why I don't want to publish to subdir of a domain. these apps were designed to be at the root of the domain. this makes the situation worse to by having an hardcoded path which is going to break once we deploy to its own domain.

Either give it it's own root domain or make this an environment variable parameter

Reply:
I did it using env variable but I put a condition saying if the variable is not set it's served to root, I am not sure if this is the right way to do since I don't know if that behavior is wanted or if the variable should always be set

Reply:
Unresolving this conversation as I see that this version doesn't use the basename variable anymore

Reply:
that is not the most recent commit I hardcoded it again in the last to second commit to debug

<img width="390" alt="Screenshot_154" src="https://github.com/ai-cfia/finesse-frontend/assets/9827730/8a8823c3-e710-49c0-9193-47b70e7651ef">